migrations/20240910065643-create-company.js

```diff
@@ -1,34 +0,0 @@
-'use strict';
-/** @type {import('sequelize-cli').Migration} */
-module.exports = {
-  async up(queryInterface, Sequelize) {
-    await queryInterface.createTable('companies', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      name: {
-        type: Sequelize.STRING(120),
-        allowNull:false
-      },
-      email: {
-        type: Sequelize.STRING(50),
-        allowNull:false
-      },
-      gender: {
-        type: Sequelize.ENUM("male","female","other"),
```

Comment:
Declare enum on top of file as such 
enum GenderEnum {
 Male,
 Female,
 Other
}